client/components/repository/Workflow/Status.vue

```diff
@@ -1,20 +1,20 @@
 <template>
-  <v-select v-on="$listeners" v-bind="$attrs" clearable>
+  <v-select v-on="$listeners" v-bind="$attrs">
```

Comment:
Since there's no more `Filters` context coming from a parent directory, should we change the name of this component to `SelectStatus`?